packages/workflow/src/NodeHelpers.ts

```diff
@@ -1035,19 +1153,23 @@ export function getParameterValueByPath(nodeValues: INodeParameters, parameterNa
  * @param {string} path The path to the properties
  * @returns {INodeIssues}
  */
-export function getParameterIssues(nodeProperties: INodeProperties, nodeValues: INodeParameters, path: string): INodeIssues {
+export function getParameterIssues(
+	nodeProperties: INodeProperties,
+	nodeValues: INodeParameters,
+	path: string,
+): INodeIssues {
 	const foundIssues: INodeIssues = {};
 	let value;
 
 	if (nodeProperties.required === true) {
 		if (displayParameterPath(nodeValues, nodeProperties, path)) {
 			value = getParameterValueByPath(nodeValues, nodeProperties.name, path);
 
-			if (nodeProperties.typeOptions !== undefined && nodeProperties.typeOptions.multipleValues !== undefined) {
+			if (nodeProperties.typeOptions?.multipleValues !== undefined) {
```

Comment:
This was automatic? Beautiful.

Reply:
unfortunately not 🙈 but we are emitting a warning if this pattern is not applied 👍🏼